### PR TITLE
Add Navbar Component

### PR DESCRIPTION
**Pull request by varun13169**

### CODE REVIEW

File: components/navbar/navbarPage.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    margin-bottom: 2rem;
+}
+
+.container__code-preview {
+    width: 80rem;
+    min-height: 22rem;
+    margin-top: 5rem;
+    align-self: center;
+    justify-content: center;
+    overflow: scroll;
```
Comment (zoef-yt) [inline components/navbar/navbarPage.css]: Add auto from now so it would be easy for future

**Review by zoef-yt — CHANGES_REQUESTED** [review]

File: components/navbar/navbarPage.css (hunk 1 above)
Comment (varun13169) [reply components/navbar/navbarPage.css]: Done.

**Review by zoef-yt — APPROVED** [review]
Good code. Good to merge.

**Comment by humzahasan** [comment]
image

Not sure if it's related to this PR or not, but do look into this. Carbon frame has a scroll as well as the outer container.

**Comment by varun13169** [comment]
> image
> 
> Not sure if it's related to this PR or not, but do look into this. Carbon frame has a scroll as well as the outer container.

That's a known issue @humzahasan, will be fixing it in a single PR.

**Review by humzahasan — APPROVED** [review]
Looks Good.

**Pull request merged**